### PR TITLE
fix: reference an asset from public if that doesn't change often

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

There is a bug that images that referencing 'src/assets/**' dynamically with variables.
To solve this problem, there are two solutions.

1. move assets to public and fix the path to reference public assets.
2. make vite to bundle it in build time. follow [this issue](https://github.com/vitejs/vite/issues/1265).

**I think in this case, the browser images are fixed, they must always there and they don't change much.**
It's inefficient to check every build time.
So I choose the first solution.

### CODE REVIEW

File: apps/web/src/common/modules/modals/RecommendedBrowserModal.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
                    target="_blank"
                 >
                     <img class="browser-img"
-                         :src="`src/assets/images/${browser.img}`"
+                         :src="browser.img"
```
Comment (skdud4659) [inline apps/web/src/common/modules/modals/RecommendedBrowserModal.vue:19]: question: why don't we use `srcset`?

**Review by skdud4659 — APPROVED** [review]
LGTM!

File: apps/web/src/common/modules/modals/RecommendedBrowserModal.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/common/modules/modals/RecommendedBrowserModal.vue:19]: Oh I didn't know this. Thanx for information! 👍

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Review by sulmoJ — APPROVED** [review]
LGTM~!

**Pull request merged**